packages/wallet-adapter-core/src/WalletCore.ts

```diff
@@ -83,6 +83,8 @@ import { aptosStandardSupportedWalletList } from "./AIP62StandardWallets/registr
 
 export type IAptosWallet = AptosStandardWallet & Wallet;
 
+export type AnyAptosWallet = Wallet | AptosStandardSupportedWallet;
```

Comment:
could we add a comment explaining what this type is for

Reply:
Just added a JSDoc to the type

Reply:
nit: this can live under a `type` file https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-core/src/AIP62StandardWallets/types.ts